system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
```

Comment:
Man, thanks for finding this -- mystery freakin' solved. That was driving me *crazy* last week and I couldn't find the culprit, though I did suspect a difference in the setup of the dev system and the sys-int system. But even with that intuition, I still couldn't track this down.